server/api/api.go

```diff
@@ -124,3 +128,74 @@ func (h *Handler) setLink(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(status)
 	_, _ = w.Write([]byte(`{"status": "OK"}`))
 }
+
+func (h *Handler) deleteLink(w http.ResponseWriter, r *http.Request) {
+	autolinkName := r.URL.Query().Get(autolinkclient.AutolinkNameQueryParam)
+	if autolinkName == "" {
+		h.handleError(w, errors.New("autolink name or pattern should not be empty"))
+		return
+	}
+
+	links := h.store.GetLinks()
+	found := false
+	for i := 0; i < len(links); i++ {
+		if links[i].Name == autolinkName || links[i].Pattern == autolinkName {
+			links = append(links[:i], links[i+1:]...)
+			found = true
+		}
+	}
+
+	status := http.StatusNotModified
+	if found {
+		if err := h.store.SaveLinks(links); err != nil {
+			h.handleError(w, errors.Wrap(err, "unable to save link"))
+			return
+		}
+		status = http.StatusOK
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(status)
+	_, _ = w.Write([]byte(`{"status": "OK"}`))
+}
+
+func (h *Handler) getLinks(w http.ResponseWriter, r *http.Request) {
+	links := h.store.GetLinks()
+
+	autolinkName := r.URL.Query().Get(autolinkclient.AutolinkNameQueryParam)
+	if autolinkName == "" {
+		h.handleSendingJSONContent(w, links)
+		return
+	}
+
+	found := false
+	var autolinks []autolink.Autolink
+	for i := range links {
+		if links[i].Name == autolinkName || links[i].Pattern == autolinkName {
+			autolinks = append(autolinks, links[i])
+			found = true
+		}
+	}
+
+	if !found {
+		h.handleError(w, errors.Errorf("no auto link found with name or pattern %s", autolinkName))
```

Comment:
```suggestion
		h.handleError(w, errors.Errorf("no auto-link found with name or pattern %s", autolinkName))
```

Reply:
why this change autolink is a single word I don't think this is necessary

Reply:
okay, just remove the space than